wsd/COOLWSD.cpp

```diff
@@ -4959,6 +4959,10 @@ void dump_state()
 #if !MOBILEAPP
     Admin::dumpMetrics();
 #endif
+
+    std::lock_guard<std::mutex> docBrokerLock(DocBrokersMutex);
```

Comment:
I assume and hope that this lock ordering is consistent everywhere =)

Reply:
I maintained the same order, yes. And it seems these signal-triggered functions are the only place we need to take both locks. I didn't revise the logic, though not a fan of multi-locks.